spec/docker/image_spec.rb

```diff
@@ -318,6 +318,19 @@
         expect(image.info[:headers].keys).to include 'X-Registry-Auth'
       end
     end
+
+    context 'with a block capturing create output' do
+      let(:create_output) { "" }
+      let(:block) { Proc.new { |chunk| create_output << chunk } }
+      let!(:image) { subject.create('fromImage' => 'hawknewton/true', &block) }
```

Comment:
I used `hawknewton/true` because the test removes the image in an `after` block and I didn't want to collide with other specs.  I've since learned that a pull will always emit the message I'm looking for, so it should be safe to use `tianon/true` and remove the `after` hook.

Thanks, I'll update the PR.